features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/item-manage/js/jquery.certUi.js

```diff
@@ -0,0 +1,305 @@
+             * */
+            jagg.post("/site/blocks/item-design/ajax/add.jag",
+                {
+                    action: "addClientCertificate",
+                    alias: alias,
+                    certificate: certString,
+                    name : name,
+                    version : version,
+                    provider : provider,
+                    tierName : tierName
+                },
+                function (result) {
+                    if (!result.error) {
+                        if (result.message.code !== 1) {
+                            jagg.message({
+                                content: getMessage(result.message),
+                                type: "error"
+                            });
+                            return;
+                        }
+                        certUi.config.certificates.push(cert_payload);
+                        jagg.message({
+                            content: getMessage(result.message),
+                            type: "info"
+                        });
+                        certUi._on_modal_close();
+                        certUi.render();
+                    } else {
+                        jagg.message({
+                            content: getMessage(result.message),
+                            type: "error"
+                        });
+                    }
+                }, "json");
+        }
+    };
+
+    $.fn[name] = function (options) {
+        return this.each(function () {
+            if (!$.data(this, "plugin_" + name)) {
+                $.data(this, "plugin_" + name,
+                    new CertUi(this, options));
+            }
+        });
+    };
+
+    /**
+     * Returns specific message based on the response code.
+     * 1: Success
+     * 2: Internal Server error
+     * 3: Certificate exists in trust store.
+     * 4: Certificate not found.
+     * */
+    var getMessage = function (msgObject) {
+        switch (msgObject.code) {
+            case (1) : {
+                if (msgObject.action === "add") {
+                    return i18n.t("The certificate was added successfully.");
+                } else {
+                    return i18n.t("The certificate was deleted successfully.")
+                }
+            }
+            case (2) : {
+                if (msgObject.action === "add") {
+                    return i18n.t("Failed to add Certificate due to an Internal Server Error.");
+                } else {
+                    return i18n.t("Failed to delete Certificate due to an Internal Server Error.");
+                }
+            }
+            case (3) : {
+                return i18n.t("Failed to add Certificate in to Publisher Trust Store. Certificate exists for" +
+                    " the Alias");
+            }
+            case (4) : {
+                return i18n.t("Failed to delete the certificate. Certificate could not found for the given alias." +
```

Comment:
Shall we rephrase this as "Failed to delete the certificate. Certificate could not **be** found for the given alias. " The highlighted word is missing,